frappe/public/js/frappe/widgets/onboarding_widget.js

```diff
@@ -68,17 +75,48 @@ export default class OnboardingWidget extends Widget {
 		return $step;
 	}
 
+	go_to_page(step) {
+		let current_route = frappe.get_route();
+
+		frappe.set_route(step.path).then(() => {
+			let msg_dialog = frappe.msgprint({
+				message: __(step.callback_message),
+				title: __(step.callback_title),
+				primary_action: {
+					action: () => {
+						frappe.set_route(current_route).then(() => {
+							this.mark_complete(step);
+						});
+						msg_dialog.hide();
+					},
+					label: () => __("Continue"),
```

Comment:
I didn't understand the workflow.

The msgprint will appear on the loading of the page. How, can I complete or go back immediately without exploring the page. Even though if I do not need to change anything on the page (chart of accounts), still I need to explore the CoA before taking the judgement whether I need to make an changes or not.